x-pack/plugin/security/cli/build.gradle

```diff
@@ -31,11 +31,7 @@ tasks.named("forbiddenPatterns").configure {
 }
 
 tasks.named("thirdPartyAudit").configure {
-  ignoreMissingClasses(
-    // Used in org.bouncycastle.pqc.crypto.qtesla.QTeslaKeyEncodingTests
-    'junit.framework.Assert',
-    'junit.framework.TestCase'
-  )
+  ignoreMissingClasses()
```

Comment:
not 100% sure why this empty ignoreMissingClasses() is needed. In 8.x I was able to remove the custom configuration entirely. Without this it will blow up thinking that there classes that missing but don't have any issues.  "All excluded classes seem to have no issues." . I think this is subtle bug in the custom logic only for 7.x in the thirdparty audit and/or forbiden API plugins.

Reply:
This does not make big difference even if kept, but I was curious and tested it. Removing `ignoreMissingClassses` did not fail my local build. 
Unless I'm missing something, this was the build command I run after removal: `./gradlew :x-pack:plugin:security:cli:build`. It completed successfully.

Reply:
WTF !  I can remove it now and it works fine. Thanks for double checking...must have been user error or some gradle caching issue. removed on [`a69ce85` (#103818)](https://github.com/elastic/elasticsearch/pull/103818/commits/a69ce85cfc2a81910105f99c05d0a3e14b2562ac)